src/core/TabPanel/TabPanel.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import Typography from '@material-ui/core/Typography';
+import { useTheme } from '@material-ui/core/styles';
+import { makeStyles } from '@material-ui/core/styles';
+
+interface TabPanelProps {
+  children?: React.ReactNode;
+  index: any;
+  value: any;
+}
+
+const TabPanel: React.FC<TabPanelProps> = ({
+  children,
+  index,
+  value,
+  ...other
+}) => {
+  const useStyles = makeStyles({
+    tabPanel: {
+      padding: useTheme().spacing(1.5),
+    },
+  });
```

Comment:
Can we send this over to `styles.ts` then we could also access `theme` instead of `useTheme().spacing`